main.go

```diff
@@ -113,6 +113,12 @@ func ParseArgs() (string, []string) {
 		Help(subcommand)
 	}
 
+	// list command can have no arguments since it can use the config from login
+	// so we immediately return in that case
+	if command == "list" {
+		return command, os.Args
+	}
```

Comment:
```suggestion
```

otherwise this changes the general behavior pattern that subcommand without arguments prints the subcommand's usage. See comment in code  right below.

Reply:
Yes but now list command can work without arguments since we it can work with the .sda-cli-session as a config from a previous login.

Reply:
so I should update the usage and readme I guess.

Reply:
oh right, then yes :-)

Reply:
added